.github/workflows/create-release-build.yaml

```diff
@@ -0,0 +1,72 @@
+# This starter workflow is for a CMake project running on a single platform.
+name: Create Release Build
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+  # Manual trigger using the Actions page.
+  workflow_dispatch:
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build_release:
+    runs-on: ubuntu-22.04-8core
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: 'recursive'
+        ref: '${{github.ref}}'
+
+    - name: Install Dependencies
+      run: sudo apt-get -y install libunwind-dev libgflags-dev libssl-dev libelf-dev protobuf-compiler libzstd-dev
+
+    - name: Configure CMake
+      # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
+      # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
+      run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} -DENABLE_TOOL=LLVM -S ${{github.workspace}}
```

Comment:
I think this will default to using g++ as the compiler. Is that desired?

Reply:
This can be built with clang toolchain too, either works. Except for clang, we need to install necessary packages in the previous step.

Reply:
I think we should build the release binaries with clang to minimize surprises. Also the runner should already have clang-13 [1] on it so it's not necessary to install clang as a package.

https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md

Reply:
Thanks. I will add -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ and test build on my personal repo.